ios/packages/test-utils/Sources/ui-test/FlowManagerView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  FlowManagerView.swift
+//  Pods
+//
+//  Created by Zhao Xia Wu on 2023-11-01.
+//
+
+import SwiftUI
+
+/**
+ SwiftUI View to wrap the `ManagedPlayer` and handle the result
+ for use in UI testing
+ */
+public struct FlowManagerView: View {
```

Comment:
This should probably go in the demo app as well, since its tightly coupled to `ReferenceAssetsPlugin`